src/Comment.jsx

```diff
@@ -304,4 +302,4 @@ _Comment.defaultProps = {
     locale: 'eng'
 };
 
-export default translate('translations')(_Comment);
\ No newline at end of file
+export default translate('translations', {i18n: getI18nInstance()})(_Comment);
```

Comment:
Explicitly pass the "correct" instance. This avoids situations when this component translation is being broken by another component's.